srcpkgs/shadowsocks-rust/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'shadowsocks-rust'
+pkgname=shadowsocks-rust
+version=1.23.4
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="libzstd-devel"
+short_desc="Rust port of shadowsocks, a fast lightweight obfuscated tunnel proxy"
+maintainer="JkktBkkt <apkabikov@gmail.com>"
+license="MIT"
+homepage="https://github.com/shadowsocks/shadowsocks-rust/"
+distfiles="https://github.com/shadowsocks/shadowsocks-rust/archive/refs/tags/v${version}.tar.gz"
+checksum=8a91836256989e3a56409d0e83da6549ecf727e2d6642cd4e707993d9c8a23d3
+
+system_accounts="_shadowsocks"
+make_dirs="/etc/${pkgname} 0750 _shadowsocks _shadowsocks"
```

Comment:
don't use `pkgname` variable here and in `post_install`, they are not variable and would need to be changed if `pkgname` changes.